lighthouse-cli/test/smokehouse/test-definitions/dobetterweb/dbw-expectations.js

```diff
@@ -31,6 +31,14 @@ const expectations = {
       id: 'wordpress',
     }],
     MainDocumentContent: /^<!doctype html>.*DoBetterWeb Mega Tester.*aggressive-promise-polyfill.*<\/html>[\r\n]*$/s,
+    CSSUsage: {
+      stylesheets: [
+        {'header': {'sourceURL': 'http://localhost:10200/dobetterweb/dbw_tester.css?delay=2000&async=true'}},
+        {'header': {'sourceURL': 'http://localhost:10200/dobetterweb/dbw_tester.html'}},
+        {'header': {'sourceURL': 'http://localhost:10200/dobetterweb/dbw_tester.html'}},
+        {'header': {'sourceURL': 'http://localhost:10200/dobetterweb/dbw_tester.html'}},
```

Comment:
I'm having trouble here because the exact stylesheets tracked by this gatherer varies across runs, I suppose due to the delays?

1) ideas on how to make it consistent (is this a bug?)
2) ideas on alternative way to test this PR?